src/components/Profile/ProfileLink/ProfileLink.css

```diff
@@ -3,13 +3,16 @@
   align-items: center;
   gap: 0.5em;
   padding: 0.125em;
-
-  span {
-    font-size: 14px;
-  }
 }
 
 .profile__header__link a {
+  display: flex;
+  align-items: center;
   text-decoration: none;
   color: inherit;
 }
+
+.profile__header__link a span {
```

Comment:
The suggested change to edit the CSS and re-use CSS nesting was to do something along those lines:
```css
.profile__header__link {
  ...

  a {
    ...
  }

  span {
    ...
  }
}
```